src/System.Net.Requests/tests/System.Net.Requests.Tests.csproj

```diff
@@ -10,7 +10,8 @@
     <ProjectGuid>{851A40FE-7F07-415D-8592-5FE2096E84D3}</ProjectGuid>
```

Comment:
Delete
```
   <PropertyGroup>
     <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
   </PropertyGroup>
```
above in this project and that should eliminate your bin-clashing issue.

When you removed the Unix build you introduced another build that didn't define OSGroup and that property cased this to be set to OSGroup=Windows_NT. Which then ends up conflicting with your other build configuration in the .builds file (i.e. the one with net463 TestTFMs). The bin clash occurs because both the TargetGroup and the OSGroup are the same but the OSGroup is passed in as a global property in the builds file in the one case and in the other case it isn't. The output locations only take ConfigurationGroup, OSGroup, and TargetGroup into account so if they end up being the same you get a bin-clash.